nhd/NHDScheduler.py

```diff
@@ -556,3 +594,19 @@ def run(self):
                 for n, v in self.nodes.items():
                     if n == item["node"]:
                         v.SetGroups(item["groups"])
+
+           # Node addition event
+            elif item["type"] == NHDWatchTypes.NHD_WATCH_TYPE_TRIAD_NODE_CREATE:
```

Comment:
This results in a node that is cordoned in NHD, but may or may not be marked as unschedulable by kubernetes (i.e. may or may not have node.kubernetes.io/unschedulable:NoSchedule taint applied).  In our nominal workflow, we add nodes and cordon them, but there definitely have been bugs where that doesn't work right that could result in a node getting added uncordoned.  In this case, NHD workloads wouldn't get scheduled to the node, but there would be no indication of why on the node itself (i.e. no taint/label), and we'd need to scour NHD logs to figure out what's up.

Would it make sense to taint or label the node to promote visibility NHD's internal active/inactive state (e.g. apply a sigproc.viasat.io/nhdActiveNode=False label or something)?